### ISSUE TITLE
public source code

### ISSUE THREAD

**Issue by therealssj** [gitscout/gitscout-feedback#264]
will the source code be made public at any point so we can modify it a little for personal use?

Thnx for your work so far! Great tool!

**Comment by jeremybenaim** [comment]
Hey @therealssj, as of today there is no plan to make the code public, sorry! 

Thanks for the kind words! 💛

**Comment by nijikokun** [comment]
Please publish the source since you're shutting down.

**Comment by osheroff** [comment]
agreed, we'd love to maintain this project.  It's a pity to let it go to waste.
